graphql-test/src/main/java/org/jahia/test/graphql/GraphQLSchedulerTest.java

```diff
@@ -95,6 +95,12 @@ public void testJobSubscription() throws Exception {
                                     "}\n" +
                                 "}";
 
+        // try authenticating first before connecting to websocket
+        httpClient.createGet(getBaseServerURL() + Jahia.getContextPath() + "/jahia")
+                .addHeader("Authorization", "Basic " + Base64.getEncoder().encodeToString("root:root1234".getBytes()))
+                .addHeader("Origin", getBaseServerURL())
+                .build().execute();
+
         String url = (getBaseServerURL() + Jahia.getContextPath() + "/modules/graphql").replaceFirst("http", "ws");
         WebSocketClient webSocketClient = httpClient.createWebSocket(url)
```

Comment:
We should remove the AuthorizationHeader from the websocket to be sure, at worst we shoul;d transfer the JSESSIONID cookie from the previous response to the websocket